blogs.json

```diff
@@ -6152,6 +6152,25 @@
             "site_url": "https://www.twitch.tv/dfreniche",
             "feed_url": "https://twitchrss.appspot.com/vod/dfreniche",
             "twitter_url": "https://twitter.com/dfreniche"
+          },
+          {
+            "title": "MoureDev en Twitch",
+            "author": "MoureDev",
+            "site_url": "https://www.twitch.tv/mouredev",
```

Comment:
Would it be possible to add a `feed_url` here please? It's a mandatory field for the CI to complete. Just copy the format of the `https://twitchrss.appspot.com/vod/xxxxxxxxx` URLs used for the other Twitch channels.

Thanks!